fv3core/stencils/fvtp2d.py

```diff
@@ -86,10 +86,10 @@ def __init__(self, namelist, hord):
         self.stencil_q_i = stencil_wrapper(q_i_stencil)
         self.stencil_q_j = stencil_wrapper(q_j_stencil)
         self.stencil_transport_flux = stencil_wrapper(transport_flux_xy)
-        self.xppm_inner = XPPM(spec.namelist, ord_inner)
-        self.yppm_inner = YPPM(spec.namelist, ord_inner)
-        self.xppm_outer = XPPM(spec.namelist, ord_outer)
-        self.yppm_outer = YPPM(spec.namelist, ord_outer)
+        self.xppm_inner = XPiecewiseParabolic(spec.namelist, ord_inner)
+        self.yppm_inner = YPiecewiseParabolic(spec.namelist, ord_inner)
+        self.xppm_outer = XPiecewiseParabolic(spec.namelist, ord_outer)
+        self.yppm_outer = YPiecewiseParabolic(spec.namelist, ord_outer)
```

Comment:
The new guidelines say these attributes should also be plainly named, can you use something like `x_piecewise_parabolic_inner`?